cmd/robot/master.go

```diff
@@ -88,7 +88,7 @@ func (v *masterVerb) Run(ctx context.Context, flags flag.FlagSet) error {
 	}
 	tempDir := file.Abs(tempName)
 	restart := false
-	serverAddress := fmt.Sprintf("localhost:%v", v.Port)
+	serverAddress := fmt.Sprintf(":%v", v.Port)
```

Comment:
This fixes remote uploads to master. It could be split to another CL if needed, but it is mentioned in the CL description.